Build race with API headers on Appveyor

Recently I've seen this a few times on Appveyor:
https://ci.appveyor.com/project/DynamoRIO/dynamorio/builds/30344395
```
13686====> FAILURE in debug-internal-64 <====
13687debug-internal-64: **** 1 build errors ****
13688	C:\projects\dynamorio\build\build_debug-internal-64\include\dr_api.h(47) : fatal error C1083: Cannot open include file: 'dr_defines.h': No such file or directory
13689final package: **** 85 build errors ****
13690	ntdll_imports_x64.def : error LNK2001: unresolved external symbol __isascii
13691	ntdll_imports_x64.def : error LNK2001: unresolved external symbol __iscsym
13692	ntdll_imports_x64.def : error LNK2001: unresolved external symbol __iscsymf
13693	ntdll_imports_x64.def : error LNK2001: unresolved external symbol __toascii
13694	ntdll_imports_x64.def : error LNK2001: unresolved external symbol _atoi64
```

A re-build makes it go away.  I think it's a race introduced recently -- and my theory is drdisas is missing a dependence.  I will fix it.

--- a/clients/drdisas/CMakeLists.txt
+++ b/clients/drdisas/CMakeLists.txt
@@ -35,6 +35,7 @@ include(../../make/policies.cmake NO_POLICY_SCOPE)
 add_executable(drdisas drdisas.cpp)
 configure_DynamoRIO_decoder(drdisas)
 use_DynamoRIO_extension(drdisas droption)
+add_dependencies(drdisas api_headers)
 
 if (WIN32)
   append_property_string(TARGET drdisas COMPILE_FLAGS "/EHsc")

